`IntoSystemSetConfigs::chain_ignore_deferred` has wrong return type

## Bevy version

53910e07aefc40a9f9ca089f643d731b748c699f (Bevy 0.13.2)

## What you did

```rust
#[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
struct SetA;

#[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
struct SetB;

fn main() {
    App::new()
        .configure_sets(Update, (SetA, SetB).chain_ignore_deferred());
}
```

## What went wrong

`IntoSystemSetConfigs::chain_ignore_deferred`'s return type is not the same as `IntoSystemSetConfigs::chain`. Because of that you can't use `chain_ignore_deferred` with system sets

Compile error:
```
error[E0277]: the trait bound `NodeConfigs<std::boxed::Box<dyn bevy::prelude::System<In = (), Out = ()>>>: IntoSystemSetConfigs` is not satisfied
   --> src/main.rs:30:33
    |
30  |         .configure_sets(Update, (SetA, SetB).chain_ignore_deferred())
    |          --------------         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `SystemSet` is not implemented for `NodeConfigs<std::boxed::Box<dyn bevy::prelude::System<In = (), Out = ()>>>`, which is required by `NodeConfigs<std::boxed::Box<dyn bevy::prelude::System<In = (), Out = ()>>>: IntoSystemSetConfigs`
    |          |
    |          required by a bound introduced by this call
    |
```

--- a/crates/bevy_ecs/src/schedule/config.rs
+++ b/crates/bevy_ecs/src/schedule/config.rs
@@ -721,6 +721,10 @@ impl IntoSystemSetConfigs for SystemSetConfigs {
     fn chain(self) -> Self {
         self.chain_inner()
     }
+
+    fn chain_ignore_deferred(self) -> Self {
+        self.chain_ignore_deferred_inner()
+    }
 }
 
 impl<S: SystemSet> IntoSystemSetConfigs for S {

--- a/crates/bevy_ecs/src/schedule/config.rs
+++ b/crates/bevy_ecs/src/schedule/config.rs
@@ -650,7 +650,7 @@ where
     /// Ordering constraints will be applied between the successive elements.
     ///
     /// Unlike [`chain`](Self::chain) this will **not** add [`apply_deferred`](crate::schedule::apply_deferred) on the edges.
-    fn chain_ignore_deferred(self) -> SystemConfigs {
+    fn chain_ignore_deferred(self) -> SystemSetConfigs {
         self.into_configs().chain_ignore_deferred()
     }
 }

